docs/sections/workflows/workchain.md

```diff
@@ -613,101 +621,149 @@ Out[3]:
 
 :::
 
-## Work chain with Calculation Jobs
+## Submitting calculation jobs
 
-Until now, we have written work chains that create data via `calculation function` processes.
-These processes are executed by the same machine process that is executing the work chain.
-The full potential of a work chain is only accessed when it creates independent processes to execute the varios steps of the outline, such as running an external code.
-This is achieved through an AiiDA process called `calculation job` ({class}`~aiida.engine.processes.calcjobs.calcjob.CalcJob`).
-The execusion of the calculation jobs are managed and controlled by the AiiDA **daemons**.
+All work chains we have seen up to this point rely on calculation _functions_ to create data.
+When running the work chain, these processes are executed by the same Python process that is executing the code in the work chain methods.
+All the functionality of the work chains above could have been implemented in a work _function_.
+In fact, this would be much more simple, similar to the `add_multiply` work function shown in the {ref}`work function module <workflows-workfunction>`.
 
-Here, we demonstrate how to include calculation jobs in our work chain.
-We are going to use the example in Section {ref}`Two calculation functions and more outputs<workflows-workchain-adding-complexity>` and replace the `addition` calculation function.
-The resulting code looks like this:
+Of course, the power of a work chain lies in its ability to _submit_ other processes that can run independently while the work chain waits for them to complete.
+Doing so also releases the daemon to do other tasks, which is vital when running many workflows in high-throughput.
+These processes often don't run Python code - think for example of a remote binary running on a supercomputer.
```

Comment:
```suggestion
These processes often don't run Python code - think for example of an external code running on a supercomputer.
```

Reply:
Why "external"? Isn't remote more clear? I'd also use binary here to indicate that it's a binary code.

Reply:
Your message is also clear to me. However, I think that an "external code" is a concept that we used before in the tutorial in the context of calculation jobs. "Remote binary" just sounds a bit abstract to me.

Reply:
Yeah, fair point. Maybe just for consistency... In the schedule we also refer to "remote clusters", maybe I can use "remote codes" at least. 👍